appveyor_after_build.ps1

```diff
@@ -0,0 +1,300 @@
+    Retry $command -maxRetries 30 -retryDelay 1 -Verbose
+}
+
+function InvokeGetRestMethod
+{
+    param ([string]$query)
+
+    $request = $sonarQubeBaseUrl + $query;
+    $response = Invoke-RestMethod $request -Method Get -TimeoutSec 30 
+
+    return $response
+}
+
+function InvokePostRestMethod
+{
+    param ([string]$query, [bool]$asAdmin=$false)
+
+    $requestUrl = $sonarQubeBaseUrl +  $query;
+    Write-Host "POST $requestUrl"
+
+    if ($asAdmin)
+    {
+       $authHeader = GetSonarQubeAdminAuthHeader
+       $allheaders = @{Authorization = $authHeader}   
+    }    
+
+    $response = Invoke-RestMethod $requestUrl -Method Post -TimeoutSec 30 -Headers $allheaders
+    
+
+    return $response
+}
+
+#
+# Upload a file
+#
+# Remark: Multipart upload is not supported out of the box by powershell's Invoke-RestMethod, but System.Net.Http can do it
+#
+function InvokeUpload {
+     param ([string]$query, [string]$formFieldName, [String]$fileToUpload)
+    
+    [System.Reflection.Assembly]::LoadWithPartialName('System.Net.Http') | Out-Null
+
+    $requestUrl = $sonarQubeBaseUrl +  $query;   
+
+    try
+    {
+        $fileContent = [System.IO.File]::ReadAllText($fileToUpload);
+        $client = (New-Object System.Net.Http.HttpClient)
+        $stringContent = New-Object System.Net.Http.StringContent @(,$fileContent)
+        
+        $user = "admin"
+        $pass= "admin"
+        $pair = "$($user):$($pass)"
+        $encodedCreds = [System.Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes($pair))
+        $basicAuthValue = "Basic $encodedCreds"
+
+        $client.DefaultRequestHeaders.Add("Authorization", $basicAuthValue)
+
+
+        $formData = New-Object System.Net.Http.MultipartFormDataContent
+        $formData.Add($stringContent, $formFieldName, "UNUSED");
+
+
+        # hit it
+        return $client.PostAsync($requestUrl, $formData).Result
+        
+    }
+    finally
+    {
+        $client.Dispose()
+        $formData.Dispose()        
+    }
+}
+
+#
+# Gets the default admin:admin basic auth header that can be used in APIs that require elevation
+#
+function GetSonarQubeAdminAuthHeader
+{
+        $user = "admin"
+        $pass= "admin"
+
+        $pair = "$($user):$($pass)"
+
+        $encodedCreds = [System.Convert]::ToBase64String([System.Text.Encoding]::ASCII.GetBytes($pair))
+        $basicAuthValue = "Basic $encodedCreds"
+
+        return $basicAuthValue        
+}
+
+function UploadTestQualityProfile
+{
+    echo "Finding the quality profile on disk"
+    $qualityProfile = FindSingleFile ([System.IO.Path]::Combine($env:APPVEYOR_BUILD_FOLDER, "IntegrationTestProject")) "TestQualityProfile.xml"
+
+    $response = InvokeUpload "/api/profiles/restore" "backup" $qualityProfile
+}
+
+
+function SetTestQualityProfileAsDefault
+{
+    $response = InvokePostRestMethod "/api/profiles/set_as_default?language=cs&name=$testQualityProfileName" $true    
+}
+
+function VerifyAnalysisResults
+{
```

Comment:
Please let me know if you can think of other verifications or ways to increase the coverage of the test.